Assets/Scripts/Puzzle/SoundPuzzle.cs

```diff
@@ -1,3 +1,4 @@
+using System.Collections;
 using System.Collections.Generic;
 using System.Linq;

```

Comment:
Unused using statement

Reply:
This is required for IEnumerator since it isn't used as a generic in the context of a coroutine 👍